packages/cli/README.md

```diff
@@ -21,6 +13,15 @@ or
 ```
 $ npm install --global @dxos/cli@beta
 ```
+*Note: You will need to be logged into your npm account in the terminal for this to work*
+
+## Testing
```

Comment:
@Shro3 Take a look at the number of `#` that each title has. They are used to express hierarchy. The lower the number of `#` a title has, the higher in hierarchy it is.

For example if we have:
```
## Installation

### Extensions
```

This means that "Extensions" is a sub-section of "Installation".

Placing `## Testing` between these two, makes "Extensions" become a child of "Testing" instead of "Installation".

So we should find a way to keep that consistency. Maybe moving "Testing" right after the whole "Installation" section, or directly convert it into a child of "Installation".